model_config.yaml

```diff
@@ -0,0 +1,11 @@
+data:
+    mag:
+        year: 2000
+        metadata: ["Id", "Ti", "AA.AfId","AA.AfN","AA.AuId","AA.DAuN","AA.S","CC","D","F.DFN","F.FId","J.JId","J.JN","Pt","RId","Y","DOI","PB","BT","IA","F.FN"]
+        fos: ["disinformation", "misinformation", "fake news", "factoid", "half truth", "misinformation effect"]
+        entity_name: 'F.FN'
+        query_count: 1000
+        offset: 0
```

Comment:
`offset` can be hard-coded to zero in the script - a non-zero value wouldn't really make sense